superset/templates/appbuilder/navbar_right.html

```diff
@@ -24,15 +24,26 @@
     {% set locale = 'en' %}
 {% endif %}
 
-{% if not current_user.is_anonymous %}
+{% set can_sqlquery = appbuilder.sm.can_access('can_sqllab', 'Superset') %}
```

Comment:
looks like this is from a previous branch/commit, you should remove from this PR